[2021-08-09 14:22:03,391] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask5 2021-08-09T08:35:31.265897+00:00 [queued]>
[2021-08-09 14:22:03,401] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask5 2021-08-09T08:35:31.265897+00:00 [queued]>
[2021-08-09 14:22:03,401] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-09 14:22:03,402] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-08-09 14:22:03,402] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-09 14:22:03,508] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): Covidtask5> on 2021-08-09T08:35:31.265897+00:00
[2021-08-09 14:22:03,513] {standard_task_runner.py:52} INFO - Started process 20901 to run task
[2021-08-09 14:22:03,523] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Covid', 'Covidtask5', '2021-08-09T08:35:31.265897+00:00', '--job-id', '346', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp6mu76xx_', '--error-file', '/tmp/tmpamwgdy2o']
[2021-08-09 14:22:03,525] {standard_task_runner.py:77} INFO - Job 346: Subtask Covidtask5
[2021-08-09 14:22:03,923] {logging_mixin.py:104} INFO - Running <TaskInstance: Covid.Covidtask5 2021-08-09T08:35:31.265897+00:00 [running]> on host deepika-Inspiron-5570
[2021-08-09 14:22:04,423] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Covid
AIRFLOW_CTX_TASK_ID=Covidtask5
AIRFLOW_CTX_EXECUTION_DATE=2021-08-09T08:35:31.265897+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-09T08:35:31.265897+00:00
[2021-08-09 14:22:07,092] {python.py:151} INFO - Done. Returned value was: None
[2021-08-09 14:22:07,095] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Covid, task_id=Covidtask5, execution_date=20210809T083531, start_date=20210809T083703, end_date=20210809T083707
[2021-08-09 14:22:07,833] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-09 14:22:07,870] {local_task_job.py:151} INFO - Task exited with return code 0
